<ui version="4.0" >
 <class>GuiPrinterAutoSetupBase</class>
 <widget class="QWidget" name="GuiPrinterAutoSetupBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PrinterAutoSetup</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label_title" >
     <property name="text" >
      <string>&lt;h1>Printer Auto Setup&lt;/h1></string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QGridLayout" >
       <item row="0" column="1" >
        <widget class="QLabel" name="label_drivers" >
         <property name="text" >
          <string>Drivers</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_printers" >
         <property name="text" >
          <string>Printers / Devices:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QListWidget" name="listWidgetPrinters" />
       </item>
       <item row="2" column="1" >
        <widget class="QListWidget" name="listWidgetDrivers" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QCheckBox" name="checkBoxShowAllDevices" >
         <property name="text" >
          <string>Sho&amp;w all devices 
(not necessarily printers)</string>
         </property>
         <property name="shortcut" >
          <string>Alt+W</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDriverDetails" >
         <property name="text" >
          <string>Driver De&amp;tails</string>
         </property>
         <property name="shortcut" >
          <string>Alt+T</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer15" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="buttonHelp" >
       <property name="text" >
        <string>He&amp;lp</string>
       </property>
       <property name="shortcut" >
        <string>Alt+L</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack" >
       <property name="text" >
        <string>Bac&amp;k</string>
       </property>
       <property name="shortcut" >
        <string>Alt+K</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRestart" >
       <property name="text" >
        <string>&amp;Restart Autodetection</string>
       </property>
       <property name="shortcut" >
        <string>Alt+R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonManualSetup" >
       <property name="text" >
        <string>&amp;Manual Printer Setup</string>
       </property>
       <property name="shortcut" >
        <string>Alt+M</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave" >
       <property name="text" >
        <string>Sa&amp;ve Settings</string>
       </property>
       <property name="shortcut" >
        <string>Alt+V</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>buttonSave</tabstop>
  <tabstop>buttonHelp</tabstop>
  <tabstop>buttonRestart</tabstop>
  <tabstop>checkBoxShowAllDevices</tabstop>
  <tabstop>buttonDriverDetails</tabstop>
  <tabstop>buttonManualSetup</tabstop>
  <tabstop>buttonBack</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
